pkg/resource/alias/sdk.go

```diff
@@ -289,6 +298,12 @@ func (rm *resourceManager) sdkUpdate(
 	defer func() {
 		exit(err)
 	}()
+	if delta.DifferentAt("Spec.FunctionEventInvokeConfig") {
+		_, err = rm.syncEventInvokeConfig(ctx, desired)
+		if err != nil {
+			return nil, err
+		}
+	}
 	input, err := rm.newUpdateRequestPayload(ctx, desired, delta)
 	if err != nil {
```

Comment:
After the `if delta.DifferentAt` gets exectued the code starting from line 307 will get executed as well (even if user only changed `FunctionEventInvokeConfig`. How can you prevent that from happening?

Reply:
Have added `!delta.DifferentExcept("Spec.FunctionEventInvokeConfig")` condition in `sdkUpdate` to prevent execution of rest of the update part, when only `EventInvokeConfig` fields are changed.